cli/search/search.go

```diff
@@ -115,6 +115,64 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitFromGitLocal returns hash or pr/ID info from specified local git repo.
+func GetCommitFromGitLocal(repo string, input string) (string, error) {
+	if _, err := exec.LookPath("git"); err != nil {
+		return "", fmt.Errorf("unable to get commits: `git` command is missing")
+	}
+
+	isPullRequest, pullRequestID := util.IsPullRequest(input)
+
+	if isPullRequest {
+		commandStr := "pull/" + pullRequestID +
+			"/head:" + input
+		cmd := exec.Command("git", "fetch", "origin", commandStr)
+		cmd.Dir = repo
+		err := cmd.Run()
+		if err != nil {
+			return "", err
+		}
+	}
+
+	cmd := exec.Command("git", "show", input, "--quiet")
```

Comment:
We have `ExecuteCommandGetOutput` in `util` package. Can it be used here?

Reply:
It receives commands in bytes not as an optional argument for cmd.Write, so I can't just use this command without problems...